pkg/watermark/processor/entity.go

```diff
@@ -57,7 +57,8 @@ var _ ProcessorEntitier = (*ProcessorEntity)(nil)
 
 // _defaultKeySeparator is the key separate when we have shared OT buckets.
 // NOTE: we can only use `_` as the separator, Jetstream will not let any other special character.
-//       Perhaps we can encode the key using base64, but it will have a performance hit.
+//
```

Comment:
I think this is also unintended. It does look like a block code comment.

Reply:
Same here, 1.19 format.